modules/cnv-dedicated-resources.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-edit-vms.adoc
+// * cnv/cnv_vm_templates/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-dedicated-resources_{context}"]
+
+= Enabling dedicated resources for a {object}
+
+Use this procedure to enable dedicated resources for a {object}.
+
+When you enable dedicated resources for your {object},
+you obtain CPU time from a specific CPU or set of CPUs, maximizing efficiency
+while optimizing usage of system resources.
+
+.Procedure
+
+To enable dedicated resources, do the following:
+
+. Ensure that the `cpumanager` node label for the CPU resource you want to
+dedicate is set to `true`. If you do not do this,
+you will receive an error message when you attempt to dedicate the resource,
+indicating that no qualified nodes with the required labels (`cpumanager=true`)
+exist.
+.. Verify that CPU Manager is running on the CPU node whose
+resources you want to dedicate.
+.. Set the node policy to `static`.
```

Comment:
it's not the node policy, it is the CPU manager policy running on the node.

Reply:
Changed to "Set the CPU manager policy that is running on the node to `static`."